### ISSUE TITLE
Few issues are there, any idea

### ISSUE THREAD

**Issue by skprasadu** [andyocnz/Bonopoly#2]
I downloaded your code and followed all your instructions. The application is coming up, but when I am creating a game, following error is coming,

a. No default values for assumption_id,edition_id at insert of a game. I forcefully assigned 0, 0 for both. This created the game and I was able to create team and new users.

b. Now when I log in with new users and change change "Demand forecast" in production. For US if I change growth percentage to +value, it is throwing an error like "Illegal string offset 'est_demand' in /home/krishna/my-application/php/Bonopoly/game.php on line 2578"

Basically it is not saving any settings. 

I am using PHP version 5.6.40-57+ubuntu20.04.1+deb.sury.org+1 . Any help from you is appreciated.

Krishna
